.infra/Infra.Parameters.json

```diff
@@ -4,6 +4,23 @@
   "parameters": {
     "appServiceName": {
       "value": "qooked"
+    },
+    "cosmosDbContainers": {
+      "value": [
+        {
+          "name": "recipes",
+          "partitionKey": "/partitionKey",
```

Comment:
Is the partition key value here just a place holder for now until we map out what values we are gonna store?

Reply:
What this is saying is that we will have a field for each document in the database called "partitionKey" which will act as the partition key in the database.

Since we don't currently support users, that value for partition key will be the name of the collection. Once we add users we will change it to be the user ID.

I guess I could rename each partition key to be something like "recipePartitionKey" but it's your call. This is just how my team has it set up.